### PR TITLE
chore: move specs into their respective component folders

### PR DESCRIPTION
**Pull request by chrisdholt**
# Description
This PR creates README.md files for all components and moves the component specs to their respective folders. The readme files are not exhaustive and are not intended to be. Over time, it is expected that these files grow, but for now, we are starting fairly simple with a quick blurb about the control and then pointing to the component spec.

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/web-components/fast-components/src/anchor/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# fast-anchor
+`fast-anchor` is a web component implementation of an [HTML Anchor Element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a). The component supports the same visual appearances as the button component (accent, lightweight, neutral, outline, stealth) as well as a hypertext appearance for use inline with text.
+
+View the [component specification](../button/button.spec.md)
```
Comment (janechu) [inline packages/web-components/fast-components/src/anchor/README.md:4]: nit: I'd make all of these complete sentences so:
```
For more information, view the [component specification](../button/button.spec.md).
```

File: packages/web-components/fast-components/src/badge/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# fast-badge
+*Badge* component is used to highlight an item and attract attention or flag status, such as “New”, “Sale”, or other short phrases or a number to represent unread messages, updates available, etc..
```
Comment (janechu) [inline packages/web-components/fast-components/src/badge/README.md:2]: nit: used "or" twice, might consider rewording a little.

File: packages/web-components/fast-components/src/button/README.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# fast-button
+`fast-button` is a web component implementation of an [HTML Button Element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button). The component supports several visual apperances (accent, lightweight, neutral, outline, stealth).
```
Comment (janechu) [inline packages/web-components/fast-components/src/button/README.md:2]: nit: not sure about the capitalization of "Button" and "Element", anyone know what we should do here? It's inline so I'm inclined to think it should be "HTML button element".

File: packages/web-components/fast-components/src/divider/README.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# fast-divider
+A web component implementation of a [hr](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr).
```
Comment (janechu) [inline packages/web-components/fast-components/src/divider/README.md:2]: Should we spell out "horizontal rule"? We've done so for anchor.

File: packages/web-components/fast-components/src/text-area/README.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+# fast-text-area
+An implementation of an [HTML textarea element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea).
```
Comment (EisenbergEffect) [inline packages/web-components/fast-components/src/text-area/README.md:2]: Should this say "as a form-connected web-component" like the other descriptions?

Comment (chrisdholt) [reply packages/web-components/fast-components/src/text-area/README.md:2]: It technically has not been form connected so I haven't denoted it here. Once updated that can be added in.

File: packages/web-components/fast-components/src/anchor/README.md [hunk 6]
```diff
@@ -0,0 +1,4 @@
+# fast-anchor
+`fast-anchor` is a web component implementation of an [HTML Anchor Element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a). The component supports the same visual appearances as the button component (accent, lightweight, neutral, outline, stealth) as well as a hypertext appearance for use inline with text.
```
Comment (janechu) [inline packages/web-components/fast-components/src/anchor/README.md:2]: Following the "HTML button element" this should be "HTML anchor element".

**Pull request merged**